### PR TITLE
Add ChiSquared moment

### PR DESCRIPTION
**Pull request by patel-zeel**
Add moments and tests for the below distributions as part of #5078

* pymc.distributions.continuous.ChiSquared

### CODE REVIEW

File: pymc/tests/test_distributions_moments.py [hunk 1]
```diff
@@ -173,6 +174,21 @@ def test_beta_moment(alpha, beta, size, expected):
     assert_moment_is_expected(model, expected)
 
 
+@pytest.mark.parametrize(
+    "nu, size, expected",
+    [
+        (1, None, 1),
+        (1, 5, np.full(5, 1)),
+        (np.arange(1, 6), None, np.arange(1, 6)),
```
Comment (ricardoV94) [inline pymc/tests/test_distributions_moments.py:182]: Size we only have one parameter, it's probably fine to drop this test
```suggestion
```

**Review by ricardoV94 — COMMENTED** [review]
Thanks for opening a PR. I left a small suggestion. Looks great by the way!

**Comment by patel-zeel** [comment]
Agree. Implemented the suggestion :)

**Review by ricardoV94 — COMMENTED** [review]
Looks good. Waiting to see if the tests pass

**Review by ricardoV94 — APPROVED** [review]

**Comment by patel-zeel** [comment]
Not sure why ```test_exec_nuts_init``` failed. Can you please help me with this @ricardoV94?

**Comment by ricardoV94** [comment]
> Not sure why `test_exec_nuts_init` failed. Can you please help me with this @ricardoV94?

Can you replicate the failure locally?

**Comment by patel-zeel** [comment]
I tried and it seems to work locally.
image

**Comment by ricardoV94** [comment]
Probably it was a fluke, because find_MAP is still starting from prior draws instead of moments, so that test is not even deterministic. The changes to the Chisquared from this PR look completely unrelated. I'll merge this already.

**Comment by ricardoV94** [comment]
Thanks for your help @patel-zeel!

**Comment by patel-zeel** [comment]
Thank, you @ricardoV94 for the clarification and for accepting the PR :)

**Pull request merged**